README.md

```diff
@@ -152,7 +168,7 @@ From here, you can either EnsureAuthenticated in your pipeline, or on a per-cont
 defmodule MyApp.MyController do
   use MyApp.Web, :controller
 
-  plug Guardian.Plug.EnsureAuthenticated, on_failure: { MyApp.MyHandler, :unauthenticated }
+  plug Guardian.Plug.EnsureAuthenticated, handler: MyApp.MyAuthHandler
 end
```

Comment:
So to be clear, explicit handler like this is fine, it's the global config I am against. Otherwise looks good!